src/nvim/api/vim.c

```diff
@@ -218,7 +218,7 @@ ArrayOf(String) vim_list_runtime_paths(void)
   return rv;
 }
 
-/// Changes vim working directory
+/// Changes vim's working directory
```

Comment:
old wording was better. Maybe capitalize `Vim`. Avoid possessive case in general.